org.adempiere.base/src/org/compiere/model/MSysConfig.java

```diff
@@ -839,4 +842,20 @@ public String toString()
 			+"]";
 	}
 
+	/** Returns list of clientID with a System Configurator equals to value */

```

Comment:
The name of this method MSysConfig.getAll is misleading

But, I think is not necessary, you can drop this method, see comment below in Msg.